wordpress/wordpress-ha-aurora.yaml

```diff
@@ -256,6 +271,8 @@ Conditions:
   HasManagedPolicyArns: !Not [!Equals [!Ref ManagedPolicyArns, '']]
   HasEFSBackupRetentionPeriod: !Not [!Equals [!Ref EFSBackupRetentionPeriod, 0]]
   HasWAF: !Not [!Equals [!Ref ParentWAFStack, '']]
+  HasCloudFrontPrefixList: !Not [!Or [!Equals [!Ref AWS::Region, 'ap-northeast-3'], !Equals [!Ref AWS::Region, 'ap-southheast-3']]]
```

Comment:
does this work?

  HasCloudFrontPrefixList: !Not [!Condition HasNotCloudFrontPrefixList]
   HasNotCloudFrontPrefixList: !Equals [!FindInMap[], '']

Reply:
Yes, great idea!